src/main/java/by/teachmeskills/module/UserEntityFields.java

```diff
@@ -0,0 +1,16 @@
+package by.teachmeskills.module;
+
+public class UserEntityFields {
```

Comment:
From the name seems like there should be user entity fields (like "id", "name", etc). For queries you can create class like SqlQueries